app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,122 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
```

Comment:
I'm so glad that we were able to get this setup from the beginning! So often styling ends up getting put off and then overlooked in the final push.